src/components/advanced-color-picker/docgenInfo.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
I think we don't need docgenInfo.json anymore

Reply:
deleted.